frontend/ic_vetkeys/src/utils/utils.ts

```diff
@@ -68,6 +68,26 @@ export class TransportSecretKey {
     }
 }
 
+/**
+ * Check if a transport public key is valid
+ *
+ * This tests if the passed byte array is of the expected size and encodes
+ * a valid group element.
+ */
+export function isValidTransportPublicKey(tpk: Uint8Array): boolean {
```

Comment:
We could already make an entry in the respective CHANGELOG in some `[Unreleased]` section.